GoogleForCreators/web-stories-wp
# Updated to new tooltips

## Context

This updates the icons in the page and carousel menu to the new tooltips.

Note 1: Updating the tooltip for the keyboard shortcut dialog toggle is implemented in #6403.

## User-facing changes

Hover the icons in the page action menu (remove page, duplicate page, add page, undo, redo, play/stop animation) or in the carousel menu (open grid view, toggle meta-boxes) and see the difference from the old dark tooltips to the new bright tooltips.

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Hover the icons in the page action menu (remove page, duplicate page, add page, undo, redo, play/stop animation) or in the carousel menu (open grid view, toggle meta-boxes)
2. Observe the new bright tooltips

### UAT




## Checklist




---

Fixes #6377
